torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -785,7 +772,7 @@ def nonexisting_script():
 
         @torch.jit.script
         def rpc_async_call_remote_nonexisting_torchscript_in_torchscript(
-            dst_worker_name: str
+            dst_worker_name: str,
```

Comment:
nit: as they are just single arg, what's those comma for?

Reply:
These are auto-formatter generated changes. It always append comma for the last argument.